src/citrine/informatics/predictors.py

```diff
@@ -66,6 +67,28 @@ def get_type(cls, data) -> Type['Predictor']:
                 'Must be in {}.'.format(data['config']['type'], type_dict.keys())
             )
 
+    def _wrap_training_data(self, training_data: Optional[Union[DataSource, List[DataSource]]]) -> List[DataSource]:
+        """
+        Converts ``None`` to an empty list and wraps a single data source in a list that contains a single element.
+
+        Parameters
+        ----------
+        training_data: Optional[Union[DataSource, List[DataSource]]]
+            Either a single data source, list of data sources or ``None``
+        Returns
+        -------
+        List[DataSource]
+            A list of data sources
+        """
+        if training_data is None:
+            return []
+        if not isinstance(training_data, Iterable):
```

Comment:
`dict` is also Iterable.  Why not check `isinstance(training_data, DataSource)`?

Reply:
That's a good idea. I'll update it.